controllers/users.controller.mjs

```diff
@@ -109,29 +111,34 @@ export default function initUsersController(db) {
   //   }
   // };
 
-  const verify = async (req, res) => {
-    // check if cookies exist
+  const verifyUserIsLoggedIn = (req, res) => {
+    const loggedIn = verify(req, res);
```

Comment:
I abstracted the logic to verify user into another function because I wanted to protect the route with the `verify` function

Reply:
hey @staceyng , what's the format for the user log in cookie? i was trying to test the endpoints with postman, but kept running into the `result: false` error

Reply:
Oh right the cookies are `loggedInHash` and `user`
What i did was to do a post /api/login  so that the cookies will be in the postman request
<img width="1449" alt="image" src="https://user-images.githubusercontent.com/30279812/193178767-4751ed66-7bd3-42a2-8f19-517cf7e51ca4.png">